pkgs/build-support/setup-hooks/auto-patchelf.sh

```diff
@@ -167,9 +153,14 @@ autoPatchelf() {
     # findDependency outside of this, the dependency cache needs to be rebuilt
     # from scratch, so keep this in mind if you want to run findDependency
     # outside of this function.
-    findElfs "$prefix" | while read -r elffile; do
-        autoPatchelfFile "$elffile"
-    done
+    while IFS= read -r -d $'\0' file; do
+        if isELF "$file"; then
```

Comment:
Maybe even:

```sh
isELF "$file" || continue
readelf -l "$file" | grep -q '^ *INTERP\>' || continue
autoPatchelfFile "$file"
```

IMHO makes it less `pyramidic`.

Reply:
micro optimization...